### ISSUE TITLE
BUG - Mockserver issue with singleton entities in 2.1.22

### ISSUE THREAD

**Issue by heimwege** [SAP/open-ux-odata#220]
Hi,

when switching from 2.1.15 to 2.1.22 the browser console shows the following error after loading an object page:

```
Log.js?eval:452 2022-08-03 10:59:29.880899 Failed to read path /AuthorizationRestrictions/isNotAllowedToDelete - Error: Unsupported Action

    at Object.createError (http://localhost:3000/resources/sap/ui/model/odata/v4/lib/_Helper.js:376:15)
    at eval (http://localhost:3000/resources/sap/ui/model/odata/v4/lib/_Requestor.js?eval:1262:23)
    at Array.forEach (<anonymous>)
    at visit (http://localhost:3000/resources/sap/ui/model/odata/v4/lib/_Requestor.js?eval:1244:14)
    at eval (http://localhost:3000/resources/sap/ui/model/odata/v4/lib/_Requestor.js?eval:1312:5) sap.ui.model.odata.v4.ODataPropertyBinding
```

Seems like there's an issue with loading singleton entities (cds annotated with `@odata.singleton`, metadata: `<Singleton Name="AuthorizationRestrictions" Type="TaskManagementService.AuthorizationRestrictions"/> [...] <Annotation Term="UI.DeleteHidden"><Path>/AuthorizationRestrictions/isNotAllowedToDelete</Path></Annotation>`)

The JSON file AuthorizationRestrictions.json is loaded successfully

`info server:ux-fe-mockserver JSON file found for AuthorizationRestrictions`

Its content looks as follows:

```
[{
  "isNotAllowedToDelete": true
}]
```

### Problem
{describe the problem} 

### Fix
{describe the fix} 

### Why was it missed
{Some explanation why this issue might have been missed during normal development/testing cycle}  

### How can we avoid this
{if we don’t want to see this type of issues anymore what we should do to prevent}

**Comment by nlunets** [comment]
Hi @heimwege could you please share what is wrong in terms of odata query ? From our end the singleton should still be ok, but it really depends on what query is being triggered by the frontend code !
Thanks.

**Comment by heimwege** [comment]
@nlunets sure 😸 

```
GET AuthorizationRestrictions/isNotAllowedToDelete HTTP/1.1
Accept:application/json;odata.metadata=minimal;IEEE754Compatible=true
Accept-Language:en
Content-Type:application/json;charset=UTF-8;IEEE754Compatible=true
```

response 2.1.22
```
HTTP/1.1 500 Internal Server Error
sap-tenantid: tenant-default
content-type: text/plain
odata-version: 4.0

Unsupported Action
```

response 2.1.15
```
HTTP/1.1 200 OK
sap-tenantid: tenant-default
odata-version: 4.0
content-type: application/json;odata.metadata=minimal;IEEE754Compatible=true

{"@odata.context":"/odata/v4/TaskManagementService/$metadata#Edm.Boolean","@odata.metadataEtag":"W/\"12265-aiLfMn2qg0D79a7PYjP8qaNCwbY\"","value":true}
```

**Comment by nlunets** [comment]
Thanks, now i see what's wrong :)
Will provide a fix soon

**Comment by nlunets** [comment]
Fixed in version 2.1.24
